src/Bicep.Core/Registry/OciModuleRegistry.cs

```diff
@@ -165,29 +166,27 @@ public override async Task<bool> CheckArtifactExists(OciModuleReference referenc
             return mainLayers.Single();
         }
 
-        private void ValidateModule(OciArtifactResult artifactResult)
```

Comment:
I had assumed providers would be given their own version of this class (OciProviderRegistry) or something...?

Reply:
It's fine if this if for later.

Reply:
Upon review of that strategy, we will likely not do that and instead generalize the registry to be an `OciArtifactRegistry` that handles both providers and modules as indistinguishable `OciArtifacts`